src/ol/control/MousePosition.js

```diff
@@ -46,6 +47,8 @@ const COORDINATE_FORMAT = 'coordinateFormat';
  * initially and the last position is retained when the mouse leaves the viewport.
  * When a string is provided (e.g. `'no position'` or `''` for an empty string) it is used as a
  * placeholder.
+ * @property {boolean} [wrapX=true] Wrap the world horizontally on the projection's antimeridian, if it
```

Comment:
Looks like the current default is `false`.  Do you think it should be `true`?

Reply:
I initially meant to make it `true` by default, but then I decided to make it `false` for backwards compatibility. What would be your preference?